internal/pkg/archiver/archiver.go

```diff
@@ -248,6 +288,14 @@ func archive(workerID string, seed *models.Item) {
 
 			stats.HTTPReturnCodesIncr(strconv.Itoa(resp.StatusCode))
 
+			if globalBucketManager != nil {
+				if resp.StatusCode == 200 {
```

Comment:
should we check other 2XX code?

Reply:
Yes definitely, also technically a success can be like 404 or other like that.

Reply:
I think a failure should be 429, 5XX of course, possibly 403 but it's trickier.

Reply:
Moving to the feat/retries branch.